src/commands/lambda/flare.ts

```diff
@@ -17,15 +17,23 @@ import FormData from 'form-data'
 import inquirer from 'inquirer'
 import JSZip from 'jszip'
 
-import {API_KEY_ENV_VAR, AWS_DEFAULT_REGION_ENV_VAR, CI_API_KEY_ENV_VAR, SKIP_MASKING_ENV_VARS} from './constants'
+import {
+  API_KEY_ENV_VAR,
+  AWS_DEFAULT_REGION_ENV_VAR,
+  CI_API_KEY_ENV_VAR,
+  CI_SITE_ENV_VAR,
+  SITE_ENV_VAR,
+  SKIP_MASKING_ENV_VARS,
+} from './constants'
 import {getAWSCredentials, getLambdaFunctionConfig, getRegion} from './functions/commons'
 import {confirmationQuestion, requestAWSCredentials} from './prompt'
 import * as commonRenderer from './renderers/common-renderer'
 import * as flareRenderer from './renderers/flare-renderer'
 
 const {version} = require('../../../package.json')
 
-const ENDPOINT_URL = 'https://datad0g.com/api/ui/support/serverless/flare'
+const DEFAULT_DD_SITE = 'datadoghq.com'
+const ENDPOINT_PATH = '/api/ui/support/serverless/flare'
```

Comment:
We are never validating that the provided site is correct.

Reply:
Resolved using `isValidDatadogSite()`